11_10_2020/L_2_arr_task_7/L_2_arr_task_7/Program.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+/*given an array of integers containing n elements. For testing, provide the ability to set array 
+ * elements in different ways: when describing with initialization, assigning values (including 
+ * random ones), or entering the required values 1.Get elements that occur more than once in the 
+ * array without repeating 2. Find the 5 largest values in an integer array in one view*/
+
+namespace L_2_arr_task_7
+{
+    class Program
+    {
+        /// <summary>
+        /// Sorting our array and show the last five, becouse they are bigest from all array
+        /// </summary>
+        /// <param name="arr"></param>
+        static void Arsort(int[] arr)
+        {
+            Array.Sort(arr);
+
+            Console.WriteLine("5 highest values:");
+            for (int i = 5; i < 10; i++)
+            {
+                Console.Write(arr[i] + " ");
+            }
+        }
+
+        /// <summary>
+        /// find digit that repead in the array.
+        /// </summary>
+        /// <param name="arr_1"></param>
+        static void Same_var(int[] arr_1)
```

Comment:
first of all should be SameVar. For the second name of a methods should be a verb like FindSameNumber or GetRepeatVariables. In your case you write repeated digits in console maybe better name will be WriteRepeatVariables something like this